pkg/bpm/bpm.go

```diff
@@ -375,13 +381,18 @@ func (m *BackgroundProcessManager) GetIdentifiers() []string {
 
 // DefaultProcessBuilder returns the default process builder
 func DefaultProcessBuilder(cmd string, args ...string) *ProcessBuilder {
+	rootLogger, err := clog.NewDefaultZapLogger()
+	if err != nil {
+		stdlog.Fatal("failed to create root logger", err)
+	}
```

Comment:
ditto, please do not create another logger, acquire the logger as a parameter in the constructor.